blockchain-source/src/cardano/time.rs

```diff
@@ -26,7 +26,7 @@ impl Era {
 
     pub const SHELLEY_PREPROD: Self = Self {
         first_slot: 86400,
-        start_epoch: 0,
+        start_epoch: 4,
```

Comment:
What was the issue here? I don't remember this being mentioned anywhere (just asking to understand what it fixes)

Reply:
epoch is not correct

Reply:
current epoch is 81 but the code was saying it's 77 or sth

Reply:
I added the test to illustrate, you can refer to any block in preprod and check the epoch calculation

Reply:
Okay, so shifting the number from `0` to `4` corrects it then. Is it okay to do it though? The first `epoch` on `preprod` is https://preprod.cardanoscan.io/epoch/0, so it's `0`. Then why do we want to monitor them starting at `4`? 

> current epoch is 81 but the code was saying it's 77 or sth
Are we sure we are not skipping something during the sync?

Just making sure there's nothing

Reply:
https://preprod.cexplorer.io/epoch

Reply:
epoch 4 works correctly, there's a test for it

Reply:
first tx to our bridge goes in epoch 29 just in case https://preprod.cexplorer.io/tx/edb5323bc1c7b93e7a6b4cc3bbf7ee0d3561c8d2afa5c2a3d98de9d90afcf6e7

Reply:
Ah okay, I didn't check the content of the epochs, just wanted to make sure we don't skip anything further while syncing. 

Thank you for clarifying @gostkin!